apps/svelte.dev/content/blog/2025-01-01-whats-new-in-svelte-january-2025.md

```diff
@@ -0,0 +1,113 @@
+- The loose parser mode creates a more modern AST for developer tools to consume (**5.13.0**, [Docs](https://svelte.dev/docs/svelte/svelte-compiler#parse), [#14691](https://github.com/sveltejs/svelte/pull/14691))
+- A new "worker" exports condition helps better support bundling for worker-based environments (**5.15.0**, [#14779](https://github.com/sveltejs/svelte/pull/14779))
+
+
+_New/Updated Syntax_
+
+- `<svelte:boundary>` is a new special element that can capture errors that occur from within its subtree during client rendering (**5.3.0**, [Docs](https://svelte.dev/docs/svelte/svelte-boundary), [#14211](https://github.com/sveltejs/svelte/pull/14211))
+- #each is now supported without `as` (**5.4.0**, [Docs](https://svelte.dev/docs/svelte/each), [#14396](https://github.com/sveltejs/svelte/pull/14396))
+- The new `Spring` and `Tween` classes in `svelte/motion` make it even easier to build custom animation (**5.8.0**, [Docs](https://svelte.dev/docs/svelte/svelte-motion), [#11519](https://github.com/sveltejs/svelte/pull/11519))
+- The new `outro` option in `unmount` will play transitions before the component is removed from the DOM. (**5.13.0**, [Docs](https://svelte.dev/docs/svelte/svelte#unmount), [#14540](https://github.com/sveltejs/svelte/pull/14540))
+- The `$inspect.trace` rune causes the surrounding function to be traced in development. Any time the function re-runs as part of an effect or a derived, information will be printed to the console about which pieces of reactive state caused the effect to fire (**5.14.0**, [Docs](https://svelte.dev/docs/svelte/inspect#inspect.trace()), [#14290](https://github.com/sveltejs/svelte/pull/14290))
+
+
+_Reactivity APIs_
+
+- `createSubscriber` creates reactive values that depend on subscriptions (**5.7.0**, [Docs](https://svelte.dev/docs/svelte/svelte-reactivity#createSubscriber), [#14422](https://github.com/sveltejs/svelte/pull/14422))
+- The reactive `MediaQuery` class creates a media query and provides a current property that reflects whether or not it matches (**5.7.0**, [Docs](https://svelte.dev/docs/svelte/svelte-reactivity#MediaQuery), [#14422](https://github.com/sveltejs/svelte/pull/14422))
+- Getters and setters can now be bound to with `bind` (**5.9.0**, [Docs](https://svelte.dev/docs/svelte/bind), [#14307](https://github.com/sveltejs/svelte/pull/14307))
+- The `svelte/reactivity/window` module exports reactive versions of various `window` values, each of which has a reactive current property that you can reference in reactive contexts (**5.11.0**, [Docs](https://svelte.dev/docs/svelte/svelte-reactivity-window), [#14660](https://github.com/sveltejs/svelte/pull/14660))
+
+
+## What's new in SvelteKit, Svelte CLI and Language Tools
+
+- Vite 6 is now supported in SvelteKit - `create` templates in the CLI have been updated to this version (**kit@2.9.0**, **cli@0.6.6** [Docs](https://vite.dev/blog/announcing-vite6), [#12270](https://github.com/sveltejs/kit/pull/12270))
```

Comment:
This has been reverted (at least the cli part) https://github.com/sveltejs/cli/releases/tag/sv%400.6.9